src/thor/bidirectional_astar.cc

```diff
@@ -257,6 +257,8 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
     // We can set is_dest incorrectly in the second case, but it is the rare case.
     // The result path will be correct, because there are cosing.Allowed calls inside recost_forward
     // function in second time.
+    // TODO(nils): Should we really check the time restrictions for invariant type? The time is most
+    // likely vastly outdated at this point when not tracking it with invariant
     if (!costing_->Allowed(meta.edge, false, pred, tile, meta.edge_id, localtime,
                            time_info.timezone_index, restriction_idx) ||
         costing_->Restricted(meta.edge, pred, edgelabels_forward_, tile, meta.edge_id, true,
```

Comment:
it seemed wrong to me to check Allowed & Restricted with a valid time along the expansion in the case of an `invariant` time tracking

Reply:
more a personal preference sort of thing apparently